stdlib/format.ml

```diff
@@ -366,28 +380,28 @@ let format_pp_token state size = function
     if state.pp_current_indent != state.pp_margin - state.pp_space_left
     then pp_skip_token state
 
-  | Pp_break (n, off) ->
+  | Pp_break { no_break; yes_break = _, off, _ as yes_break; } ->
     begin match Stack.top_opt state.pp_format_stack with
     | None -> () (* No open box. *)
     | Some { box_type; width } ->
       begin match box_type with
       | Pp_hovbox ->
         if size > state.pp_space_left
```

Comment:
I think you need to add the length of the first string of yes_break to size here.

Reply:
Could we get a regression test against this?